spec/models/web_results_post_processor_spec.rb

```diff
@@ -68,7 +68,7 @@
                          :published_at => DateTime.parse("2011-09-26 21:33:06"),
                          :guid => '80799 at www.whitehouse.gov',
                          :rss_feed_url_id => rss_feed_urls(:white_house_blog_url).id)
-        15.times do |x|
```

Comment:
I think this change is not necessary. See my comments in https://rrsoft.slack.com/archives/G8APZCEH3/p1531181738000177